ethcore/res/ethereum/foundation.json

```diff
@@ -4888,6 +4888,105 @@
 				}
 			}
 		},
+		"000000000000000000000000000000000000000a": {
+			"builtin": {
+				"name": "bls12_381_g1_add",
+				"activate_at": "0xffffffff",
```

Comment:
The convention is usually set this as `0x7fffffffffffff`. Any distant-future number works just fine though.